### ISSUE TITLE
`org-roam-protocol-open-file` fails due to file path escaping

### ISSUE THREAD

**Issue by krobbins** [jethrokuan/org-roam#483]
### Description
I am new to org-protocol, so I may not have all the pieces configured properly. But, when I click on a node in the org-roam graph in Firefox `org-roam-protocol-open-file` fails due to the file path still being escaped for backslashes and spaces.

I was able to get this working by changing
```
(find-file file)
```
to
```
(find-file (org-link-unescape file))
```
in `org-roam-protocol-open-file`.

### Versions
- Emacs (`C-h v emacs-version`): v26.3
- Org-roam commit: https://github.com/jethrokuan/org-roam/commit/6175739b33b0abaa862c0137e216e6029f24d5cd

**Comment by jethrokuan** [comment]
On what file path is this failing on? I tried a file with spaces and backslashes and it still opens fine.

**Comment by krobbins** [comment]
I apologize for opening an issue prematurely. I was having trouble on files with spaces and colons, e.g. 2020-04-17T12:31-0600_what is the problem.org. I tracked the problem down to a configuration error. I had `(require org-protocol)` in my use-package `:init` for org-roam instead of the proper `(require org-roam-protocol)`. Navigating from the graph now works.

When I edited and loaded the modified `org-roam-protocol-open-file` I was actually enabling the org-roam protocols. What confused me was that it /almost/ worked.
